_content/articles/viewpoint-are well-intended-buddhist-practices_everard-et-al.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Viewpoint: Are well-intended Buddhist practices an under-appreciated threat to global aquatic biodiversity?"
```

Comment:
Is "Viewpoint: " necessary? I took that as like... the section of the magazine or whatever...